### ISSUE TITLE
Comparison of the different assemblers

### ISSUE THREAD

**Issue by chochart** [MicroB3-IS/osd-analysis#31]
Hi everyone,

I'm actually working on metagenomic assemblies of ocean microbiom and I would be interested to know how did you compare the different assemblers and make your choice. Do you provide any access to the comparison of the different results? Or enlighten me on your work? 

Thanks,
Corentin

**Comment by genomewalker** [comment]
Hi Corentin,
I need to upload all the comparisons we did. I used a combination of [metaQUAST](http://bioinf.spbau.ru/metaquast), [ALE](http://sc932.github.io/ALE/) and for a few samples we had long reads and followed a similar approach than the one described [here](http://www.nature.com/nature/journal/v464/n7285/full/nature08821.html) 
A the time when we did the assemblies, SPAdes was the assembler giving the best results. I repeated the assemblies recently with the last version of MEGAHIT and I got similar results to the SPAdes ones using less time and resources. Also you might be interested to try the recently published [MetaSPAdes](http://bioinf.spbau.ru/metaspades), I haven't benchmarked it properly but the few tests I did they look quite good.
